### PR TITLE
feat(metric): Let the user pass Wavefront URL and port via environment variables

### PR DESCRIPTION
**Pull request by aseure**
For Kubernetes!

### CODE REVIEW

File: metric_wavefront.go [hunk 1]
```diff
@@ -11,7 +13,13 @@ import (
 var wavefrontConfig *wavefront.WavefrontConfig
 
 func initMetricWavefront() error {
-	addr, err := net.ResolveTCPAddr("tcp", config.Metric.WavefrontUrl)
+	url := extractWavefrontUrl(config.Metric.WavefrontUrl)
```
Comment (sfriquet) [inline metric_wavefront.go]: nit: `WavefontURL`, Url is not a word ;p

File: metric_wavefront.go [hunk 2]
```diff
@@ -11,7 +13,13 @@ import (
...
+	if url == "" {
+		useMetric = false
```
Comment (sfriquet) [inline metric_wavefront.go]: where is this var coming from?

File: metric_wavefront.go [hunk 3]
```diff
@@ -29,6 +37,34 @@ func initMetricWavefront() error {
 	return nil
 }
 
+func extractWavefrontUrl(wavefrontUrl string) string {
```
Comment (sfriquet) [inline metric_wavefront.go]: Why not just expect the whole URL to be set in the env, in a single variable, and get rid of this method altogether?

Comment (ElPicador) [reply metric_wavefront.go]: Because kubernetes :(

File: metric_wavefront.go (hunk 1 above)
Comment (aseure) [reply metric_wavefront.go]: Indeed. Thank you.

File: metric_wavefront.go (hunk 2 above)
Comment (aseure) [reply metric_wavefront.go]: Global that is set in `owl.go`.

File: metric_wavefront.go (hunk 3 above)
Comment (aseure) [reply metric_wavefront.go]: I prefer to clearly write the environment variables names in the configuration. If at some point the name of the variables in Kubernetes change, we only need to change them in the configuration file, and not to change the hardcoded value from `owl`, push a new version, import the new version in `log-processing-go`, creating a new Docker container, etc.

TL;DR If Kubernetes changes the environment variable naming, it's faster to change it in a configuration file.

Comment (sfriquet) [reply metric_wavefront.go]: Ok, forget about the env, but why do we need to parse something that we already provide in the conf?
